src/index.js

```diff
@@ -0,0 +1,48 @@
+import './style.css';
+
+const btn = document.querySelector('form');
+const listInput = document.querySelector('#lists-input');
+const listHolder = document.querySelector('#list-holder');
+
+// *********event lister ***********
+let listArray = [];
+// Store the listArray in Local Storage when the form is submitted
+class UI {
+  static displayData() {
+    const datas = listArray.map((item) => `<div class="col-12" id="list">
+                <p> <input class="form-check-input" type="checkbox" value="" id="defaultCheck1"><span>${item.discription}</span></p>
+                <span><i class="fa fa-ellipsis-v" aria-hidden="true"></i></span>
+            </div>`);
+    listHolder.innerHTML = (datas).join(' ');
+  }
+
+  static cleanInputs() {
+    listInput.value = '';
+  }
```

Comment:
- [ ] As shown in this [video](https://www.youtube.com/watch?v=AcUd-_Yjjqg), when a user clicks on a particular to-do task there should be a line-through the tasks indicating that the task has been completed. I would kindly suggest you implement the functionality as this will show your ability to replicate any design given to you by a client and this also shows more confidence. 👍 👍